code-examples/quarkus/kafka-quickstart/README.md

```diff
@@ -0,0 +1,70 @@
+Quarkus Kafka Quickstart
```

Comment:
We don't need/want this file it's confusing. If we want the content, it needs to be integrated into our quick starts.

Reply:
Disagree. This file comes from quarkus itself and it has some info about profiles etc.

It not only lets us to check what changed but also provides quick overview for user that might not want to go thru guide

Reply:
no.

Reply:
I'm going to miss the place where I'm going to document RHOAS image, kubernetes template and env profiles for deployment. Do not want to put this into operator repository. Needs discussion

Reply:
I can remove quarkus specific content for now.